### ISSUE TITLE
add figures to repo?

### ISSUE THREAD

**Issue by Swanson-Hysell** [ian-r-rose/bayesian_plate_reconstruction#6]
While I realize that the figures can be generated via the make file it is computationally expensive to do so. Given that we will need to submit the figure files and it would be nice to see updates graphically (like the one you just processed for Australia) I think it makes sense to have them in there.

The only downside I see is that it makes the .git a bit hefty given it holding on to the versioning of those files. But this doesn't seem like a big deal to me and is something that is fine in other manuscript repos I am working with. Perhaps there are other downsides?

**Comment by Swanson-Hysell** [comment]
It also would be nice to do this so that the .tex can compile within inline figures so that we can work on revisions to captions, etc.

**Comment by ian-r-rose** [comment]
Yes, I agree completely. I initially had some lofty ideas about regenerating the figures on every build so that you always knew that they were up-to-date. But now, given that it takes hours or days to run the MCMC results, that is just not practical.

I am currently in the process of regenerating the figures using Python 3 (and the associated pickle format), and once I have those I will add them to the repo. Unfortunately, the MCMC results are too large to be stored, but the figures should be no problem.

**Comment by Swanson-Hysell** [comment]
Sounds good. Did you see my issue with the environment.yml in that pull request thread?

**Comment by ian-r-rose** [comment]
Yes, I just commented.
